Google.Api.Generator/Generation/SourceFileContext.cs

```diff
@@ -67,13 +165,18 @@ public enum ImportStyle
             { typeof(object).FullName, PredefinedType(Token(SyntaxKind.ObjectKeyword)) },
         };
 
-        public SourceFileContext(ImportStyle importStyle) => _importStyle = importStyle;
-
-        private readonly ImportStyle _importStyle;
-
-        // Namespace -> alias
-        private readonly Dictionary<string, string> _namespaceAliases = new Dictionary<string, string>();
-        private readonly HashSet<string> _namespaceAliasesOnly = new HashSet<string>();
+        public static SourceFileContext New(ImportStyle importStyle)
```

Comment:
Any reason for New rather than Create here? Not really bothered...

Reply:
Not really, changed :)